### PR TITLE
feat : add pagination plugin

### LINKED ISSUE #521: [NG] Add pagination plugin
**Desired Behavior**
Add and configure [gatsby plugin for pagination](https://www.gatsbyjs.org/packages/gatsby-awesome-pagination/) in[ layer5-ng](https://github.com/layer5io/layer5/tree/layer5-ng). Related to #481

### PR DESCRIPTION
**Pull request by Tanuj22**
**Description**

This PR fixes #521

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -124,6 +127,14 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
 
   const members = res.data.allMembers.nodes;
 
+  paginate({
+    createPage,
+    items: blogs,
+    itemsPerPage: 8,
+    pathPrefix: `/blog`,
+    component: blogViewTemplate
```
Comment (leecalcote) [inline gatsby-node.js:135]: Will we need to update this code in the future to accommodate news, workshops, and other collections? If so, is there a way to make this less fragile?

Comment (Tanuj22) [reply gatsby-node.js:135]: The paginate function would need to be called for each of the collections in that case

Comment (leecalcote) [reply gatsby-node.js:135]: Ok, a centrally-controlled configuration to enable pagination on specific collections. This makes sense.

**Comment by Tanuj22** [comment]
Added pagination for blogs and when filtered by tags. Things like jumping to a particular page, the last page, etc. can be done. I can come back to that later.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**